folly/Portability.h

```diff
@@ -205,8 +205,10 @@ FOLLY_GCC_DISABLE_NEW_SHADOW_WARNINGS
  */
 #if defined(_MSC_VER)
 #define FOLLY_TLS __declspec(thread)
-#elif defined(__GNUC__) || defined(__clang__)
+#elif (defined(__GNUC__) || defined(__clang__)) && !_LIBCPP_VER 
```

Comment:
Why is this a library consideration rather than a compiler consideration?

Reply:
Resolved with my updated DInkimware headers. good catch.